cmd/kubeadm/app/apis/kubeadm/v1beta2/conversion.go

```diff
@@ -64,3 +67,14 @@ func Convert_v1beta2_JoinConfiguration_To_kubeadm_JoinConfiguration(in *JoinConf
 	out.NodeRegistration.ImagePullPolicy = corev1.PullIfNotPresent
 	return nil
 }
+
+// Convert_v1beta2_BootstrapToken_To_v1_BootstrapToken is required so that
```

Comment:
Comment seems truncated

Reply:
indeed, now i need to remember what it was about.

Reply:
updated.